docs/how-to/deploy-cluster.md

```diff
@@ -0,0 +1,143 @@
+follow [this](https://github.com/input-output-hk/iohk-ops#getting-ssh-access) instruction.
+It may take DevOps some time to give you access, ping them periodically.
+
+### Preparing branch and configuration
+The next step is you need to prepare branch and configuration for deploy.
+For example you want to deploy release branch with smaller `k`, 
+then you must create new branch based on this release one and modify `devnet` configuration in `lib/configuration.yaml` in this branch.
+Sometimes you will need to create your own configuration, however,
+most likely it will be enough to modify existing `devnet` configuration.
+Then you should push your branch on github.
+
+### Creation of tmux session
+After it you should connect to `staging` jumpserver and create tmux session for the issue within which you are going to deploy cluster:
+`tmux new -s issue-id` (for instance `tmux new -s csl228`). 
+It should be done for the two reasons: to keep state of `nix-shell` between reconnections to `staging` jumpserver
+and if something goes wrong (and it will go) you can ask devops to help you attaching to this session.
+
+To attach to this session when you reconnect to `staging` server, you can use `tmux a -t issue-id`.
+
+### Deployment and recommendations
+The next steps are described [here](https://github.com/input-output-hk/internal-documentation/wiki/Developer-clusters-HOWTO).
+Some notes and recommendations about this instruction:
+* `cardano-sl` git revision is most likely HEAD commit of your branch
+* Ask DevOps which `iohk-ops` branch you should use before deploy.
+* Specify `--bump-system-start-held-by` option for `io deploy`. 
+The passed value depends on how long will deployment take: 
+if `cardano-sl` is compiling from scratch, then it probably should be 15-20 mins, otherwise it's enough 5-10 mins. 
+If deploy won't be done utill system start then the first slots will be without blocks and system most probably will die.
+* Obtain logs from one of node after you deployed the system to make sure deployment was successful.
+
+## How to prepare installers
+
+Installer is file which can install Daedalus on your machine (`*.exe` file for Windows, `*.pkg` file for MacOS).
+
+### Which parts Daedalus consists of
+Before the process of preparing installers will be described I would like to say which parts Daedalus consists of.
+There are three parts: `launcher`, `cardano-node` and Daedalus frontend.
+`cardano-node` is like application which is launched on cluster, however, it also provides API to maintain your wallets, secret keys and etc.
+So essentially it's backend for Daedalus frontend.
+
+When you open Daedalus on your machine `launcher` is launched and it launches `cardano-node` and Daedalus frontend which uses provided API.
+So installer contains all these three parts.
+`cardano-node` and `launcher` come from [cardano-sl repository](https://github.com/input-output-hk/cardano-sl), 
```

Comment:
I would write `cardano-launcher` instead of `launcher`. `launcher` might be a bit ambiguous for some people and `cardano-launcher` is more consistent with `cardano-node`.